src/ai/susi/server/api/aaa/DeleteGroupService.java

```diff
@@ -81,7 +81,7 @@ public ServiceResponse serviceImpl(Query post, HttpServletResponse response, Aut
                 result.put("message", "Group deleted successfully");
                 return new ServiceResponse(result);
             } else {
-                throw new APIException(422, "Group doesn't exists");
+                throw new APIException(401, "Group doesn't exists");
```

Comment:
401 is used for `unauthorized` status

but here the group doesn't exist so shouldn't it be a `422` or `400`